src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1121,7 +1121,7 @@
   </Node>
   <Node Name="DeclarationExpressionSyntax" Base="ExpressionSyntax">
     <Kind Name="DeclarationExpression"/>
-    <Field Name="Declaration" Type="VariableDeclarationSyntax">
+    <Field Name="VariableComponent" Type="VariableComponentSyntax">
```

Comment:
Correct, so that (as it says on the tin)

> it can be extended in the future to allow `out *` or `out var (x, y)` etc without requiring any further changes to the syntax model.

---
In reply to: [73611277](https://github.com/dotnet/roslyn/pull/12917#discussion_r73611277) [](ancestors = 73611277)